requirements.txt

```diff
@@ -1,2 +1,2 @@
 mopti==0.10.10
-bofire @ git+https://github.com/experimental-design/bofire.git@01d301df27f5cccf5087f37ec9b43a6e12401d04
\ No newline at end of file
+bofire==0.0.12
```

Comment:
you said >= would also be an option. Why not here? Won't pinning the exact version cause us problems in other applications that might want to use both this and bofire?